web/js-src/ruffle-embed.js

```diff
@@ -37,6 +41,20 @@ export default class RuffleEmbed extends RufflePlayer {
         if (!elem.src) {
 		return false;
         }
+        if (elem.parentElement && elem.parentElement.tagName.toLowerCase() == "object") {
+        /* Only polyfill top-level objects */
+            if (elem.hasAttribute("src")) {
+                elem.removeAttribute("src");
+            }
+            elem.height = 0;
+            elem.style.height = 0;
+            elem.width = 0;
+            elem.style.width = 0;
```

Comment:
Instead of changing width/height, what about changing `style.display` to `"none"`? I'd worry that only changing width and height could still affect layout if e.g. CSS margins were set.